### PR TITLE
Added support for player aliasing

### PR DESCRIPTION
**Pull request by jteague9**
New Identity model for a player's alias and real name.

Added search field for admin pages.

### CODE REVIEW

File: app/admin.py [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 
-from app.models import Player
+from app.models import Player, Identity
 
 
 @admin.register(Player)
 class PlayerAdmin(admin.ModelAdmin):
     list_display = ('username', 'bnet_id', 'mmr', 'race')
     readonly_fields = ('created_at', 'modified_at')
+    search_fields = ('username', 'bnet_id', 'identity__alias')
```
Comment (tso) [inline app/admin.py:10]: nice

File: app/models.py [hunk 2]
```diff
@@ -57,9 +69,9 @@ class Player(models.Model):
     mmr: int = models.IntegerField()
     wins: int = models.IntegerField()
     losses: int = models.IntegerField()
-    clan: str = models.CharField(max_length=10, null=True)
+    clan: str = models.CharField(max_length=10, null=True, blank=True)
     profile_id: int = models.IntegerField()
-
+    identity = models.ForeignKey(Identity, null=True, blank=True, on_delete=models.SET_NULL)
```
Comment (tso) [inline app/models.py:74]: might as well type annotate

File: app/models.py [hunk 3]
```diff
@@ -21,6 +21,18 @@ class Race(Enum):
     RANDOM = 'Random'
 
 
+class Identity(models.Model):
```
Comment (tso) [inline app/models.py:24]: can we add `id: uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)`. I just like UUIDs more than incrementing integers

File: app/models.py [hunk 4]
```diff
@@ -21,6 +21,18 @@ class Race(Enum):
...
+    alias = models.CharField(max_length=30)
+    first_name = models.CharField(max_length=30, null=True, blank=True)
+    last_name = models.CharField(max_length=30, null=True, blank=True)
+    twitch = models.CharField(max_length=30, null=True, blank=True)
```
Comment (tso) [inline app/models.py:28]: maybe https://docs.djangoproject.com/en/2.2/ref/models/fields/#urlfield?

**Review by tso — APPROVED** [review]

**Review by WilliamStewart — APPROVED** [review]

**Pull request merged**